packages/wallet/backend/src/auth/service.ts

```diff
@@ -54,21 +57,44 @@ export class AuthService implements IAuthService {
     return user
   }
 
+  public async resendVerifyEmail({
+    email
+  }: resendVerifyEmailArgs): Promise<void> {
+    const user = await this.userService.getByEmail(email)
+
+    // TODO: Prevent timing attacks
+    if (!user) {
+      throw new Unauthorized('Invalid account')
+    }
+    const token = getRandomToken()
+    await this.userService.resetVerifyEmailToken({
+      email,
+      verifyEmailToken: hashToken(token)
+    })
+
+    await this.emailService.sendVerifyEmail(email, token).catch((e) => {
```

Comment:
because the error is caught here, the response to frontend will be that email was successfully send
is this expected?